### DISCUSSION TITLE
Ionic Android: Connection refused

### DISCUSSION THREAD

**Discussion by ViniciusAtaide** [ionic-team/capacitor#3319] [Q&A]
I'm trying to setup my former cordova app, but no matter what option I choose the message keeps the same.

I'm running this command `ionic cap run android -l --ssl --external`

```json
{
  "appId": "br.com.medicinia.valsa",
  "appName": "Valsa",
  "bundledWebRuntime": true,
  "npmClient": "npm",
  "webDir": "www",
  "plugins": {
    "SplashScreen": {
      "launchAutoHide": true,
      "androidScaleType": "CENTER_CROP",
      "splashFullScreen": true,
      "showSpinner": true,
      "splashImmersive": true,
      "backgroundColor": "#000000"
    }
  }
}
```

this is my capacitor file.

**Comment by jcesarmobile** [comment]
--ssl flag uses a self signed certificate and the webview doesn’t load self signed certificates.
Not sure if it’s possible to use a proper certificate, but I would recommend not using --ssl param

**Reply by imhoffd** [reply]
Maybe we can detect SSL issues and print a helpful message in the web view?

**Comment by ViniciusAtaide** [comment]
it happens without ssl too
